src/actions/actionTypes.js

```diff
@@ -9,3 +9,8 @@ export const ADD_TARGET_FAILED = 'ADD_TARGET_FAILED';
 // TOPICS
 export const LOAD_TOPICS_SUCCESS = 'LOAD_TOPICS_SUCCESS';
 export const LOAD_TOPICS_FAILED = 'LOAD_TOPICS_FAILED';
+
+// SECTIONS
+export const NEW_TARGET = 'NEW_TARGET';
```

Comment:
Hmm these are not actions, I was thinking something along the lines of the following and it should go in constants file.
```
export const sectionTypes = {
  newTarget: 'newTarget',
  aboutTaget: 'aboutTarget',
  welcome: 'welcome'
}
```